manifests/k/KDE/ruqola/1.6.1/KDE.ruqola.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: KDE.ruqola

```

Comment:
Does this have to follow a specific pattern? Generally we use `org.kde.ruqola` as indentifier for such things

Reply:
I believe it has to follow a patter.

Looking at: https://docs.microsoft.com/en-gb/windows/package-manager/package/manifest?tabs=minschema%2Cinstaller-example#minimal-required-schema

and the regex in the YAML schema: https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.0.0/manifest.version.1.0.0.json#L9

it seems that the format is indeed mandatory. Other KDE packages already in winget also follow this format:
https://github.com/microsoft/winget-pkgs/blob/master/manifests/k/KDE/Dolphin/21.04.3/KDE.Dolphin.yaml

Reply:
Before it was `ARP`Publisher.`ARP`DisplayName (ARP = `A`dd or `R`emoves `P`rograms Entry)
Now we can do ShortPublisher.ShortDisplayName since it is not linked to the Publisher and DisplayName field.
The case with a few Packages is `ShortPublisher.ShortDisplayName.Branch` or `ShortPublisher.ShortDisplayName.Branch.SubBranch`